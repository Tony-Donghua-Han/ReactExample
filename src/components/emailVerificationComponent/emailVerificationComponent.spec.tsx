import React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import EmailVerificationComponent from "./emailVerificationComponent";

describe("EmailVerificationComponent", (): void => {
    let wrapper: ShallowWrapper;
    beforeEach((): void => {
        wrapper = shallow(<EmailVerificationComponent />);
    });

    test("should render without crushing", (): void => {
        expect(wrapper.debug()).toMatchSnapshot();
    });
});
