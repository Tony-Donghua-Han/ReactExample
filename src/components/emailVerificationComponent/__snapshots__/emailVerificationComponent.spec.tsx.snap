// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmailVerificationComponent should render without crushing 1`] = `
"<div className=\\"emailVerificationComponent\\" id=\\"emailVerificationComponent\\">
  <div className=\\"title\\">
    Thank you
  </div>
  <div className=\\"divider\\" />
  <div className=\\"textGroup\\">
    <div className=\\"verifiedIcon\\" />
    <div className=\\"verificationText\\">
      Your email has been verified
    </div>
  </div>
  <div className=\\"btnHolder\\">
    <button className=\\"btn\\" type=\\"button\\">
      Get Started ‚ùØ
    </button>
  </div>
</div>"
`;
